<!-- https://husarion.com/tutorials/ros-tutorials/8-unknown-environment-exploration/ -->
<launch>
    <node pkg="frontier_exploration" type="explore_client" name="explore_client" output="screen"/>

    <node pkg="frontier_exploration" type="explore_server" name="explore_server" output="screen">
        <param name="frequency" type="double" value="1.0"/>
        <param name="goal_aliasing" type="double" value="0.5"/>
        <rosparam ns="explore_costmap" subst_value="true" file="$(find mbot_nav)/param/explore_1/exploration.yaml" command="load" />
    </node>

</launch>
